imports/plugins/core/orders/client/components/lineItems.js

```diff
@@ -51,97 +66,314 @@ class LineItems extends Component {
-              </strong> :
-              <strong>
-                {this.calculateTotal(uniqueItem.variants.price, uniqueItem.shipping.rate, 0)}
-              </strong>
-            }
+  renderPopOverChildren() {
+    const { togglePopOver, editedItems, handleSelectAllItems, selectAllItems, selectedItems,
+      uniqueItems } = this.props;
+    return (
+      <div className="invoice-popover">
+          <div className="invoice-popover-controls">
+            <div className="invoice-popover-checkbox">
+              <Checkbox
+                className="checkbox"
+                checked={selectAllItems || selectedItems.length === uniqueItems.length}
+                onChange={(e) => handleSelectAllItems(e, uniqueItems)}
+              />
+            </div>
+            <div className="invoice-popover-selected">
+              { this.showLoader() }
+            </div>
+            <div className="invoice-popover-close">
+              <Button
+                className="rui btn btn-default flat icon-only pull-right"
+                icon="fa-2x fa fa-times"
+                bezelStyle="flat"
+                onClick={() => togglePopOver()}
+              />
+            </div>
+          </div>
+          {this.renderLineItems()}
+          {!_.isEmpty(editedItems) ? this.renderLineItemRefund() : null}
+          <div className="invoice-actions">
+            <div className="invoice-action-cancel">
+              <Button
+                className="pull-right"
+                bezelStyle="solid"
+                status="default"
+                label="Cancel"
+                onClick={() => togglePopOver()}
+              />
+            </div>
+            <div className="invoice-action-refund">
+              <Button
+                className="pull-right"
+                bezelStyle="solid"
+                status="primary"
+                label="Refund Items"
+              />
+            </div>
           </div>
-        </div>
-        <br/>
       </div>
     );
   }
 
-  render() {
-    const { uniqueItems, isExpanded, onClose } = this.props;
+  renderPopOver() {
     return (
       <div>
-        {uniqueItems.map((uniqueItem) => {
-          if (!isExpanded(uniqueItem._id)) {
-            return (
-              <div key={uniqueItem._id}> { this.renderLineItem(uniqueItem) } </div>
-            );
-          }
+        <Popover
+          isOpen={this.props.popOverIsOpen}
+          attachment="middle center"
+          targetAttachment="middle center"
+         // {/*buttonElement={<Button bezelStyle="flat" label="My Button" tagName="button"/>}*/ } //
+          showDropdownButton={false}
+        >
+          {this.renderPopOverChildren()}
+        </Popover>
+      </div>
+    );
+  }
 
-          return (
-            <div className="roll-up-invoice-list" key={uniqueItem._id}>
-              <div className="roll-up-content">
+  // renderLineItemInvoice(uniqueItem) {
```

Comment:
Everything on line items will be removed when I merge in #2377. These are Michael's ticket changes